python_modules/libraries/dagster-fivetran/dagster_fivetran/resources.py

```diff
@@ -608,16 +635,42 @@ def list_connectors_for_group(self, group_id: str) -> Sequence[Mapping[str, Any]
                 break
         return results
 
-    def get_schema_config_for_connector(self, connector_id: str) -> Mapping[str, Any]:
+    def get_schema_config_for_connector(
+        self, connector_id: str, raise_on_not_found_error: bool = True
```

Comment:
Add the `raise_on_not_found_error` bool to avoid the new behavior in other callsites.